javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,143 @@
+//fullname
+function getFullname(firstname,surname){
+    return firstname+' '+surname;
+}
+const fullname = getFullname('Bill','Gates');
+console.log(fullname);
+
+//use formalname with full names
+function getFullname(firstname,surname,useFormalName){    
+    if (useFormalName == true){
+        useFormalName = 'Lord';    
+    } else { 
+        useFormalName = '';
+    }
+ return useFormalName+' '+firstname+' '+surname;
+}
+const fullname1= getFullname('Susan','cleeve', true);
+const fullname2= getFullname('Mark', 'wiens',false);
+console.log(fullname1);
+console.log(fullname2);
+
+//Event day
+let day = new Date();
+let today = day.getDay();
+const sevenDays = ['Sunday','Monday', 'Tuesday','Wednesday','Thursday','Friday','Saturday'];
+function getEventWeekday(eventInDays){
+    let eventDay = (today+eventInDays)%7;
+   return sevenDays[eventDay];   
+}
+console.log(getEventWeekday(11)); //if today is wednesday, logs out sunday.
+console.log(getEventWeekday(2)); //if today is wednesday, logs out Friday.
+
+//Weather Wear
+
+function getWhatToWear(temperature){
+    if(temperature>=17){
+        return "shorts and t-shirt"
+    }else if (temperature>=10){
+        return "Sweater and jeans"
+    }else if (temperature>=0){
+        return "winter jacket and wool hat"
+    }else{
+        return "winter jacket,boots, scarf,hand gloves and wool hat"
+    }
+}
+const clothesToWear = getWhatToWear(0);
+console.log(`As per the Weather outside and suggested clothes to wear are ${clothesToWear}.`);
+
+// classstudents
+
+const class16Students = [];
+function addStudentToClass(studentName) {  
+if(studentName === "") {
+    console.log('You cannot add an empty string to a class');
+}else if (class16Students.includes(studentName)){
+    console.log(`Student ${studentName} is already in the class.The same person cannot be added to the class.`)
+}else if (class16Students.length<6){
+  class16Students.push(studentName);}
+else if(class16Students.length=6){
```

Comment:
Do you mean class16Students.length>=6

Reply:
sorry Ida, i meant class16students==6 here, plz correct me if am wrong, what my thought is if  .length==6 means, if 7th student is  queen or if queen is already there, me allowing that student..! and after that logging out no more students....is it right or am i missing some logic? Tq